core/loadpoint.go

```diff
@@ -1450,6 +1473,42 @@ func (lp *LoadPoint) Update(sitePower float64, cheap bool, batteryBuffered bool)
 		err = lp.setLimit(targetCurrent, required)
 	}
 
+	// WakeUp checks
+	if (!lp.vehicleWakeUpCalled || !lp.chargerWakeUpCalled) && mode != api.ModeOff && lp.chargeCurrent > 0 && lp.vehicleSoc < 99 && lp.chargePower == 0 && lp.enabled && lp.activeDuration.Seconds() > 30 {
+		lp.log.DEBUG.Printf("sleeping? Mode:%s LPStatus:%s Active:%.2fm SOC:%f Current:%f Power:%f ", mode, lp.status, lp.activeDuration.Minutes(), lp.vehicleSoc, lp.chargeCurrent, lp.chargePower)
+		// check if wakup called
+		if !lp.vehicleWakeUpCalled {
+			// call the Vehicle WakeUp if available
+			if vs, ok := lp.vehicle.(api.AlarmClock); ok {
+				if err := vs.WakeUp(); err == nil {
+					lp.log.DEBUG.Printf("vehicle WakeUp API called")
+					lp.vehicleWakeUpCalled = true
+				}
+			} else {
+				// to avoid that the API will be checked every run if the veicle don't support the WakeUp call
+				lp.vehicleWakeUpCalled = true
```

Comment:
das lässt sich sicher im timer kapseln

Reply:
hab die Logik etwas angepasst aber will auch nicht jedesmal den Timer aufrufen ohne das er was machen soll.

Reply:
Super- ich schau nochmal in Ruhe durch

Reply:
ist erledigt